From e2ebd5a963612aec0f9035c7603739cf8245ce59 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 2 May 2017 10:57:53 +0100
Subject: [PATCH] v2v: -o rhv: Add Windows 2016 Server type in OVF output
 (RHBZ#1447202).

Thanks: Kun Wei

See also:
https://github.com/oVirt/ovirt-engine/commit/d233325d7b808e8bae83645ac645dfea9dfe8407

(cherry picked from commit d0344f3522c116d3ccba6d52cd100562ee6e1e23)
---
 v2v/OVF.ml | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/v2v/OVF.ml b/v2v/OVF.ml
index 1f838f543..e380d35f6 100644
--- a/v2v/OVF.ml
+++ b/v2v/OVF.ml
@@ -194,9 +194,13 @@ and get_ostype = function
     "windows_10"
 
   | { i_type = "windows"; i_major_version = 10; i_minor_version = 0;
-      i_arch = "x86_64" } ->
+      i_arch = "x86_64"; i_product_variant = "Client" } ->
     "windows_10x64"
 
+  | { i_type = "windows"; i_major_version = 10; i_minor_version = 0;
+      i_arch = "x86_64" } ->
+    "windows_2016x64"
+
   | { i_type = typ; i_distro = distro;
       i_major_version = major; i_minor_version = minor; i_arch = arch;
       i_product_name = product } ->
-- 
2.13.4

